src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaService.kt

```diff
@@ -122,7 +160,20 @@ class BehandlingFaktaService(
         return null
     }
 
-    private fun mapDokumentasjon(søknad: SøknadBarnetilsyn, grunnlagsdata: Grunnlagsdata): FaktaDokumentasjon {
+    private fun mapDokumentasjonTilSynBarn(søknad: SøknadBarnetilsyn, grunnlagsdata: Grunnlagsdata): FaktaDokumentasjon {
```

Comment:
mapDokumentasjon burde også kunne gjøres likt for begge dokumentasjonsobjektene

Reply:
Fixed: 132ad9b34613d8dd53844f0f529283cd75c5770c